test/test_broken.jl

```diff
@@ -8,22 +8,22 @@ using Miniball
 
     pts = [-2.628921573512486897783446693210862576961517333984375000000000000000000000000000 4.345646543473985268235537660075351595878601074218750000000000000000000000000000; -7.783166805400283383420401150942780077457427978515625000000000000000000000000000e-01 -4.950583883652715400103261345066130161285400390625000000000000000000000000000000; -7.110427483983996488348111597588285803794860839843750000000000000000000000000000 9.078042051418130364837111301312688738107681274414062500000000000000000000000000e-01; -2.781077447672829272562466940144076943397521972656250000000000000000000000000000 -5.261543462775741808457041770452633500099182128906250000000000000000000000000000; -6.913805339049894804759333055699244141578674316406250000000000000000000000000000 1.448828316446829855834721456631086766719818115234375000000000000000000000000000]
 
-    @test_throws ErrorException miniball(Float64.(pts))
+    #@test_throws ErrorException miniball(Float64.(pts))
     ball = miniball(Float64.(pts), check=false)
     ball_radius = sqrt(ball.squared_radius)
     pt = pts[1,:]
-    @test_broken norm(pt - ball.center) <= ball_radius * 1.4
-    @test_broken norm(pt - ball.center) <= ball_radius
+    @test norm(pt - ball.center) <= ball_radius * 1.4
```

Comment:
You can delete the test with the 1.4 was just to show that the error is that bad. Same for the 1.7 below.